scripts/track-dependency-status.sh

```diff
@@ -0,0 +1,48 @@
+# Get total count of python dependencies and outdated python dependencies
+
+total_python_deps="$(pip list| wc -l)"
+outdated_python_deps="$(pip list --outdated| wc -l)"
+
+# Get outdated JS dependency count
+
+COUNT_OUTDATED_DEP_CODE=$(cat <<-END
+import json
+import sys
+
+dep_arr = json.load(sys.stdin)
+print(len(dep_arr["data"]["body"]))
+
+END
+)
+
+outdated_js_deps="$(yarn outdated --json | sed -n 2p | python -c "$COUNT_OUTDATED_DEP_CODE")"
```

Comment:
Does this `yarn` command work on your system? I get:

```bash
$ yarn outdated --json
{"type":"error","data":"Outdated lockfile. Please run `yarn install` and try again."}
{"type":"info","data":"Visit \u001b[1mhttps://yarnpkg.com/en/docs/cli/outdated\u001b[22m for documentation about this command."}
```
I tried executing `yarn install --frozen-lockfile` but that didn't help.

Reply:
This is my fault. I haven't updated the lock file while updating a dependency which is resulting in this error. 
So `lockfile` is not in sync with the dependencies. For the sake of testing this PR I have added the `lockfile` changes in this PR itself. I am going to remove it and create a separate PR for the `lockfile` itself. 
However, you can run `yarn install` which should update your lockfile, and then run the command `yarn outdated --json`. 
Let me know if that helps.